CHANGELOG.md

```diff
@@ -4,6 +4,17 @@
 <!-- For unreleased changes, see entries in .chloggen -->
 <!-- next version -->
 
+## [0.108.0] - 2024-08-28
+
+This Splunk OpenTelemetry Collector for Kubernetes release adopts the [Splunk OpenTelemetry Collector v0.108.0](https://github.com/signalfx/splunk-otel-collector/releases/tag/v0.108.0).
+
+### 💡 Enhancements 💡
+
+- `agent`: Add a feature gate `useControlPlaneMetricsHistogramData` ([#1372](https://github.com/signalfx/splunk-otel-collector-chart/pull/1372))
+  This feature gate allows to gather control plane metrics and send them as histogram data to Observability Cloud.
+  This is an experimental feature under heavy development.
+  
+
```

Comment:
```suggestion
  This is an experimental feature under heavy development.

```

Reply:
This caused [a check to fail](https://github.com/signalfx/splunk-otel-collector-chart/actions/runs/10599312782/job/29374041977?pr=1426).